framework/helpers/BaseHtml.php

```diff
@@ -147,7 +147,7 @@ public static function tag($name, $content = '', $options = [])
         if ($name === null || $name === false) {
             return $content;
         }
-        $html = "<$name" . static::renderTagAttributes($options) . '>';
+        $html = '<' .$name . static::renderTagAttributes($options) . '>';
```

Comment:
Concatenation is less readable and slower. Why change that?

Reply:
In this case concatenation is more readable i think. There is no mix of concat and quotes, and seems more clear that brackets surround content.
And concatenation faster and use less memory.

Reply:
>  And concatenation faster and use less memory.

Nope.
https://jonczyk.me/2016/10/23/phpyths-buster-great-string-performance-test/

Reply:
@rob006 It's really not that important, but

> Conclusion
> My tests prove that it doesn’t matter if you use single or double quotes both in PHP 5.6 and 7. In concatenation test usage of dots and variable inside string were very close to each other, so I call it even.

Reply:
@schmunk42 Yes, but if it is less readable (at least for me) and slower, why change that? Performance would be only valid reason for such change, but this is not the case.

Reply:
I also thought it's more readable and no mix of quotes and there's no difference in slower/faster (but I really don't wanna debate this :) in this particular case) - it's fine. But mostly I don't care about this change.

When using double-quotes I usually enclose variables in `{}`, so what's about? :)

    "<{$name} " . static::renderTagAttributes($options)} . ">"